gramps/gen/utils/docgen/odstab.py

```diff
@@ -335,7 +335,8 @@ def open(self,filename):
 
         try:
             self.content_xml = tempfile.mktemp()
-            self.f = open(self.content_xml,"wb")
+            with open(self.content_xml,"wb") as self.f:
```

Comment:
This is going to behave differently. Before it was open, and now it will be closed. Is that right?